### ISSUE TITLE
Subadult charactermenu silhouettes no longer light up for species without a subadult stage

### ISSUE THREAD

**Issue by Hypno123** [TheIsleGame/bugs#590]
## Expected Behavior

The subadult silhouettes should light up on the charactermenu when past a certain growth of the juvenile stages for species that do not have a subadult stage.

## Current Behavior

The subadult silhouettes for Carno and Utah no longer light up when past a certain growth. The light goes from juvenile to adult.

**Notice in the following images that both juveniles are at 100% but the subadult silhouette isn't lit up.**

![image](https://user-images.githubusercontent.com/30550499/54437309-40a1f600-472c-11e9-8d74-dc6305a5fae4.png)
![image](https://user-images.githubusercontent.com/30550499/54437673-0edd5f00-472d-11e9-8d12-98a54b56a0d9.png)

## Steps to reproduce

1. Spawn as Carno or Utah juvie.
2. Grow yourself.
3. Check the charactermenu. The subadult phase will not be lit up.
4. Grow to adult. Notice it skips from the juvenile to adult silhouette.

## Build Information

> **Game Version:** 0.1.53.0

> **Operating System**: Windows

> **Network Role**: Client

**Comment by dinosauriac** [comment]
Technically, since Carno/Utah don't have a specific "subadult" growth phase, they might be superfluous.

**Comment by Emanthealmighty** [comment]
I think the sub adult silhouettes should be removed if the dinosaur itself doesn't have that stage.

I had reported [this](https://github.com/TheIsleGame/bugs/issues/535) some time ago and the response to it was that the character menu shouldn't be made any more granular, so why not just get rid of the unused assets? Maybe some time in the future there'll be a way to re-incorporate them into the character screen, but for now, they kind of only cause confusion.

**Comment by Hypno123** [comment]
Agreed. Gut any sub-adult silhouettes if the stage doesn't exist for them. I'll leave the report up until that change has been made.
